package.json

```diff
@@ -29,5 +30,8 @@
     "typescript": "^5.3.3",
     "webpack": "^5.89.0",
     "webpack-cli": "^5.1.4"
+  },
+  "dependencies": {
+    "@types/echarts": "^4.9.22"
```

Comment:
Is it voluntary to have `@types/echarts` as a non-dev dependency here?

Reply:
Sorry, A mistakes introduced during my tests on the different kind of code generation